bucket/jmkvpropedit.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.5.2",
+    "description": "A batch GUI for mkvpropedit (part of MKVToolNix) written in Java",
+    "homepage": "https://github.com/BrunoReX/jmkvpropedit",
+    "license": "BSD 2-Clause \"Simplified\" License",
+    "suggest": {
+        "JDK": [
+            "java/openjdk",
+            "java/temurinjdk"
+        ]
+    },
+    "url": "https://github.com/BrunoReX/jmkvpropedit/releases/download/v1.5.2/jmkvpropedit-v1.5.2.zip",
+    "hash": "1a3998ee7ad343852b0f8e669278b03a88d546a7335dec166cead96c33fc27b9",
+    "pre_install": "$manifest.persist | ForEach-Object { if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null } }",

```

Comment:
That's quite... clever. I didn't know you could access the `$manifest` object inside the manifest itself.

Reply:
Yeah it's a quite interesting [feature](https://scoop-docs.vercel.app/docs/concepts/Pre--and-Post-install-scripts.html#variables). I first saw it used in the NirSoft bucket. (After dealing with the pain of trying to manually check if the file exists...)